pkg/actions/tools/jj/signing.go

```diff
@@ -0,0 +1,25 @@
+package jj
+
+import (
+	"strings"
+
+	"github.com/carapace-sh/carapace"
+	"github.com/carapace-sh/carapace-bin/pkg/actions/net/ssh"
+	"github.com/carapace-sh/carapace-bin/pkg/actions/os"
+)
+
+// ActionSigningKeys completes signing keys based on the user's configuration
+func ActionSigningKeys() carapace.Action {
+	return carapace.ActionExecCommand("jj", "config", "get", "signing.backend")(func(output []byte) carapace.Action {
+		backend := strings.Trim(string(output), " \t\n")
```

Comment:
Is `TrimSpace` perhaps an alternative here?
```suggestion
		backend := strings.TrimSpace(string(output))
```

Reply:
Forgot about that function, fixed